Yosemite/Yosemite/Actions/ShippingLabelAction.swift

```diff
@@ -73,5 +73,8 @@ public enum ShippingLabelAction: Action {
                                destinationAddress: ShippingLabelAddress,
                                packages: [ShippingLabelPackagePurchase],
                                emailCustomerReceipt: Bool,
-                               completion: (Result<[ShippingLabel], Error>) -> Void)
+                               completion: (Result<[ShippingLabel], Error>) -> Void,
+                               backendProcessingDelay: TimeInterval = 2.0,
+                               pollingDelay: TimeInterval = 1.0,
+                               pollingMaximumRetries: Int64 = 3)
```

Comment:
See the `ShippingLabelStore` implementation below for where these numbers come from.

Also, how cool is it that `enum` cases associated values can have a default value?